### PR TITLE
Correct some Neutron inconsistencies

### PR DESCRIPTION
**Pull request by xenolog**

### CODE REVIEW

File: pages/reference-architecture/0060-quantum-vs-nova-network.rst [hunk 1]
```diff
@@ -10,30 +10,27 @@ ability to leverage different networking technologies to power their cloud
 networking.
 
 There are various deployment use cases for Neutron. Fuel supports the most 
-common of them, called Provider Router with Private Networks. It provides each 
-tenant with one or more private networks, which can communicate with the 
-outside world via a Neutron router. This allows full routing isolation for 
-each tenant private network.
+common of them, called Per-tenant Routers with Private Networks. It provides each 
+tenant has a virtual Neutron router with one or more private networks, which can communicate with the
```
Comment (mattymo) [inline pages/reference-architecture/0060-quantum-vs-nova-network.rst]: no lines with over 80 characters. please refactor

File: pages/reference-architecture/0060-quantum-vs-nova-network.rst [hunk 2]
```diff
@@ -110,34 +109,34 @@ The most likely configuration for different number NICs on cluster nodes:
...
-least 2 NICs.
-* To deploy OpenStack using Neutron with VLAN segmentation, each node requires
-at least 3 NICs.
-
 * Neutron will not allocate a floating IP range for your tenants. After each 
   tenant is created, a floating IP range must be created. Note that this does 
   not prevent Internet connectivity for a tenant's instances, but it would 
   prevent them from receiving incoming connections. You, the administrator, 
-  should assign a floating IP network for the tenant. Below are steps you can 
+  should assign a floating IP adresses for the tenant. Below are steps you can
```
Comment (mattymo) [inline pages/reference-architecture/0060-quantum-vs-nova-network.rst]: spelling: addresses

File: pages/reference-architecture/0060-quantum-vs-nova-network.rst [hunk 3]
```diff
@@ -10,30 +10,27 @@ ability to leverage different networking technologies to power their cloud
...
 
 Neutron is not, however, required in order to run an OpenStack environment. If 
 you don't need (or want) this added functionality, it's perfectly acceptable to 
 continue using nova-network.
 
 In order to deploy Neutron, you need to enable it in the Fuel configuration. 
-Fuel will then set up an additional node in the OpenStack installation to act 
-as an L3 router, or, depending on the configuration options you've chosen, 
-install Neutron on the controllers.
+Fuel will then set up an additional Neutron components on the controllers.
```
Comment (mattymo) [inline pages/reference-architecture/0060-quantum-vs-nova-network.rst]: I think we can improve this sentence with:
Fuel sets up Neutron components on each of the controllers to act as a router in HA (if deploying in HA mode).

File: pages/reference-architecture/0060-quantum-vs-nova-network.rst (hunk 1 above)
Comment (mihgen) [inline pages/reference-architecture/0060-quantum-vs-nova-network.rst]: "It provides each tenant has a virtual Neutron.." ??
Should it be "Each tenant has a virtual Neutron.." ?

File: pages/reference-architecture/0060-quantum-vs-nova-network.rst [hunk 4]
```diff
@@ -10,30 +10,27 @@ ability to leverage different networking technologies to power their cloud
...
-* **Public network** (also known as External) network used for Internet 
-  access for all nodes. This network also may be called “external”.
+* **Public network** (also known as External network) used for Internet 
+  access for all nodes.
 * **Floating IP network** subnet within public network allocated for tenant 
-  Internet access. A DHCP server directly assigns IP addresses for this network.
+  Internet access. A Neutron server directly assigns IP addresses for this network.
   If a user delegates a floating IP address to an instance, an IP address will 
-  be assigned from this subnet.
+  be assigned from this subnet (by snat/dnat).
```
Comment (mihgen) [inline pages/reference-architecture/0060-quantum-vs-nova-network.rst]: SNAT/DNAT

File: pages/reference-architecture/0060-quantum-vs-nova-network.rst [hunk 5]
```diff
@@ -54,29 +51,31 @@ and layer 3 (network) of the OSI network model, as compared to simple layer 3
 virtualization provided by nova-network. This is the main difference between 
 the two networking models for OpenStack. Virtual networks (one or more) can be 
 created for a single tenant, forming an isolated L2 network called a 
-"private network." Each private network can support one or many IP subnets.
+"private network". Each private network can support one or many IP subnets.
 Private networks can be segmented using two different technologies:
 
 * **VLAN segmentation** Isolated tenant "private network" traffic is managed by
```
Comment (mihgen) [inline pages/reference-architecture/0060-quantum-vs-nova-network.rst]: "private network" is isolated tenant?
should it be simply "private network" traffic is managed by Neutron...

File: pages/reference-architecture/0060-quantum-vs-nova-network.rst [hunk 6]
```diff
@@ -54,29 +51,31 @@ and layer 3 (network) of the OSI network model, as compared to simple layer 3
...
 * **VLAN segmentation** Isolated tenant "private network" traffic is managed by 
-Neutron by the use of a dedicated network adapter. This network adapter must be 
-attached to a tagged network port. This network segment also must be 
-reserved only for Neutron on each host (Computes and Controllers). You should 
-not use any other 802.1q VLANs on this network for other purposes. 
-Additionally, each private network requires its own dedicated VLAN, selected 
-from a given range configured in Fuel UI. 
-
+  Neutron by the use of a dedicated network adapter. This network adapter must be 
+  attached to a untagged network port. This network segment also must be
```
Comment (mihgen) [inline pages/reference-architecture/0060-quantum-vs-nova-network.rst:60]: Can we somehow simpler rephrase the whole paragraph?
Do I understand right that private network for tenants must not be combined with any other logical network in OpenStack, and so it must be allocated on a separated NIC which no other network uses? And the port on switch must be in trunk mode to allow Neutron tag packets by himself?

Comment (mattymo) [inline pages/reference-architecture/0060-quantum-vs-nova-network.rst:60]: for VLAN yes. in our implementation it's the management network

File: pages/reference-architecture/0060-quantum-vs-nova-network.rst [hunk 7]
```diff
@@ -54,29 +51,31 @@ and layer 3 (network) of the OSI network model, as compared to simple layer 3
...
-one NIC.
-
-It is important to note that if you use tagged networks for your configuration 
-and combine multiple networks onto one NIC, you should make the Public 
-network untagged on this NIC, but it is not a requirement. Additionally, it is 
-best if you place both the Private and Admin networks on a separate NIC from
-the Public network to ensure that traffic is separated adequately.
+  require a dedicated network adapter. Neutron builds a mesh of GRE tunnels from
+  each compute node and controller nodes to every other node. Private networks
+  for each tenant make use of this mesh for isolated traffic.
```
Comment (mihgen) [inline pages/reference-architecture/0060-quantum-vs-nova-network.rst:68]: So can it can be tagged network?

Comment (mattymo) [inline pages/reference-architecture/0060-quantum-vs-nova-network.rst:68]: for GRE definitely yes it's fine

File: pages/reference-architecture/0060-quantum-vs-nova-network.rst [hunk 8]
```diff
@@ -54,29 +51,31 @@ and layer 3 (network) of the OSI network model, as compared to simple layer 3
...
+
+It is important to note:
+
+* if you use tagged networks for your configuration 
+  and combine multiple networks onto one NIC, you should make the Public 
+  network untagged on this NIC. 
+  It is not a requirement, but best for access to the Openstack Dashboard 
+  and public Openstack API.
+* Is a best if you place the Private, Admin, Public and Management networks on a 
+  separate NIC to ensure that traffic is separated adequately.
```
Comment (mihgen) [inline pages/reference-architecture/0060-quantum-vs-nova-network.rst:78]: Fuel ensures that traffic is separated. So I would remove this point at all. We can say instead that it is recommended to spread networks across multiple NICs for best network throughput.

File: pages/reference-architecture/0060-quantum-vs-nova-network.rst (hunk 6 above)
Comment (mihgen) [inline pages/reference-architecture/0060-quantum-vs-nova-network.rst:60]: What do you mean "management network"? for what - for tenants traffic??

File: pages/reference-architecture/0060-quantum-vs-nova-network.rst (hunk 8 above)
Comment (mattymo) [inline pages/reference-architecture/0060-quantum-vs-nova-network.rst:78]: Mike, that comment needs to be directed to Przmeslaw. He has illustrated
examples for 2nics, 3nics, and 4nics to show how to separate based on what
hardware you have.

File: pages/reference-architecture/0060-quantum-vs-nova-network.rst (hunk 7 above)
Comment (mihgen) [inline pages/reference-architecture/0060-quantum-vs-nova-network.rst:68]: I just want that it's commented in docs :)

Comment (xenolog) [inline pages/reference-architecture/0060-quantum-vs-nova-network.rst:68]: GRE works over IP, and not matter tagget or not  L2 segment.

File: pages/reference-architecture/0060-quantum-vs-nova-network.rst (hunk 1 above)
Comment (xenolog) [inline pages/reference-architecture/0060-quantum-vs-nova-network.rst]: fixed

File: pages/reference-architecture/0060-quantum-vs-nova-network.rst (hunk 2 above)
Comment (xenolog) [inline pages/reference-architecture/0060-quantum-vs-nova-network.rst]: fixed

File: pages/reference-architecture/0060-quantum-vs-nova-network.rst (hunk 3 above)
Comment (xenolog) [inline pages/reference-architecture/0060-quantum-vs-nova-network.rst]: fixed

File: pages/reference-architecture/0060-quantum-vs-nova-network.rst (hunk 4 above)
Comment (xenolog) [inline pages/reference-architecture/0060-quantum-vs-nova-network.rst]: fixed

File: pages/reference-architecture/0060-quantum-vs-nova-network.rst (hunk 5 above)
Comment (xenolog) [inline pages/reference-architecture/0060-quantum-vs-nova-network.rst]: fixed

File: pages/reference-architecture/0060-quantum-vs-nova-network.rst (hunk 8 above)
Comment (pgrygiel) [inline pages/reference-architecture/0060-quantum-vs-nova-network.rst:78]: I agree that the best solution is a separate NIC for each network that we should recommend, but I would like to give users a choice how many NICs  they  will use.

File: pages/reference-architecture/0060-quantum-vs-nova-network.rst [hunk 9]
```diff
@@ -54,29 +52,31 @@ and layer 3 (network) of the OSI network model, as compared to simple layer 3
...
+* You can't combine Private or Admin network with any other networks on one NIC.
```
Comment (mattymo) [inline pages/reference-architecture/0060-quantum-vs-nova-network.rst]: Use positive language (no can'ts, according to Svetlana)
Admin and Private networks must be located together on separate NIC from the other networks.

File: pages/reference-architecture/0060-quantum-vs-nova-network.rst [hunk 10]
```diff
@@ -110,34 +110,34 @@ The most likely configuration for different number NICs on cluster nodes:
...
-least 2 NICs.
-* To deploy OpenStack using Neutron with VLAN segmentation, each node requires
-at least 3 NICs.
-
 * Neutron will not allocate a floating IP range for your tenants. After each 
   tenant is created, a floating IP range must be created. Note that this does 
   not prevent Internet connectivity for a tenant's instances, but it would 
   prevent them from receiving incoming connections. You, the administrator, 
-  should assign a floating IP network for the tenant. Below are steps you can 
+  should assign a floating IP addresses for the tenant. Below are steps you can
```
Comment (mattymo) [inline pages/reference-architecture/0060-quantum-vs-nova-network.rst:118]: address not addresses

File: pages/reference-architecture/0060-quantum-vs-nova-network.rst (hunk 9 above)
Comment (xenolog) [inline pages/reference-architecture/0060-quantum-vs-nova-network.rst]: fixed

File: pages/reference-architecture/0060-quantum-vs-nova-network.rst (hunk 10 above)
Comment (xenolog) [inline pages/reference-architecture/0060-quantum-vs-nova-network.rst:118]: not only one address. I mean addresses more correct.

Comment (mattymo) [inline pages/reference-architecture/0060-quantum-vs-nova-network.rst:118]: assign floating IP addresses (remove article 'a')

**Comment by mihgen** [comment]
Thanks folks, I'm merging this. Sveta - if you have any corrections in mind, please prepare them in separated pull request on top of master.

**Pull request merged**